reactive/etcd.py

```diff
@@ -109,6 +109,7 @@ def prepare_tls_certificates(tls):
 def remove_states():
     # stale state cleanup (pre rev6)
     remove_state('etcd.tls.secured')
+    remove_state('etcd.ssl.placed')
```

Comment:
This could easily be mistaken as part of the `stale state cleanup` comment above, can we pull them apart? Either with a blank line between, or move it up above or something.